### PR TITLE
[fix issue] fix ping command fail to execute on freebsd

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: lisa/tools/ping.py [hunk 1]
```diff
@@ -46,7 +46,18 @@ def ping_async(
     ) -> Process:
         if not target:
             target = INTERNET_PING_ADDRESS
-        args: str = f"{target} -c {count} -i {interval} -O"
+        args: str = ""
```
Comment (squirrelsc) [inline lisa/tools/ping.py]: refer to Windows in echo, 1) update create to supports FreeBSD like `if "FreeBSD" in node.os.name` in executable.py. 3) create a new class for FreeBSD.

Comment (LiliDeng) [reply lisa/tools/ping.py]: update, work like a charm

File: lisa/executable.py [hunk 2]
```diff
@@ -164,6 +164,10 @@ def create(cls, node: Node, *args: Any, **kwargs: Any) -> Tool:
             windows_tool = cls._windows_tool()
             if windows_tool:
                 tool_cls = windows_tool
+        if "FreeBSD" in node.os.name:
```
Comment (squirrelsc) [inline lisa/executable.py]: use `elif`, so the logic is clearer, and avoid strange problems.

Comment (LiliDeng) [reply lisa/executable.py]: update

**Review by squirrelsc — DISMISSED** [review]

**Pull request merged**